README.md

```diff
@@ -272,6 +272,18 @@ Select your profile picture at top right of the screen, then choose  Settings >
         * Password: your Password.\
         or
         * Token: your Personal Access Tokens.
+3. For non-English jira:
+    * If your jira `issuetype` and `priority` fields are not equal to default values ("Task" and "High") fill the configuration file:
```

Comment:
I wouldn't mention particular values. Let's just say "defaults"

Reply:
moreover for Jira with not English localization it works exactly same way, but values are localized. So probably it shouldn't be mentioned at all

Reply:
I thought we would get the field names from the JIRA API and not the user. Would that be better?

Reply:
[priority](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issue-priorities/#api-rest-api-2-priority-get) and [issuetype](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issue-types/#api-rest-api-2-issuetype-get) `id's` do not have a default value for the `High` and `Task` values.  We need to get these values from the user. We get other field names from the JIRA API.

Reply:
I added the logic for `issueType` and `issuePriority`. You can take a look.